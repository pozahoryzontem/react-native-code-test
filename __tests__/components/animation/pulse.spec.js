import { create } from 'react-test-renderer';
import React from 'react';
import configureMockStore from 'redux-mock-store';
import Pulse from '../../../src/components/animation/pulse';

describe('Pulse', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const mockStore = configureMockStore();
      const store = mockStore({});
      const props = {
        pulseSize: 200,
        startSize: 100,
        duration: 10,
      };
      const component = create(<Pulse
        store={store}
        pulseSize={props.pulseSize}
        startSize={props.startSize}
        duration={props.duration}
      />);
      expect(component.toJSON()).toMatchSnapshot();
    });
  });
});
