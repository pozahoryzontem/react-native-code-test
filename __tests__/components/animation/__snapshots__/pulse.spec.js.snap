// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pulse Rendering should match to snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "position": "absolute",
      },
      Object {
        "height": 200,
        "marginLeft": -100,
        "marginTop": -100,
        "width": 200,
      },
    ]
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#72b003",
        "borderColor": "#72b003",
        "borderRadius": 100,
        "borderWidth": 2,
        "height": 100,
        "opacity": 0.3,
        "width": 100,
      }
    }
  />
</View>
------------HTML PREVIEW---------------
<View style="0:[object Object];1:[object Object];">
  <View style="background-color:#72b003;border-color:#72b003;border-radius:100px;border-width:2px;height:100px;opacity:0.3;width:100px;"></View>
</View>
`;
